model/os/registry.yaml

```diff
@@ -11,45 +11,63 @@ groups:
       - id: os.type
         type:
           members:
+            - id: windowsnt
+              value: 'windowsnt'
+              brief: "Microsoft Windows NT based OS"
+              stability: development
             - id: windows
               value: 'windows'
               brief: "Microsoft Windows"
+              deprecated: Should use `windowsnt` instead'
               stability: development
             - id: linux
               value: 'linux'
               brief: "Linux"
+              deprecated: Should use `unix` instead'
+              stability: development
+            - id: unix
```

Comment:
I'm not sold on putting all unixes together. I think there is still value in separating `linux`, `darwin`, and `bsd` at least. While it's true they are all unix-likes, they are unique enough and are treated differently enough in the minds of most users to deserve their own attributes. I think moving all the `*bsd` attributes to `os.name` is good so we can keep that, but I think this should restore `linux`, `darwin`, and `bsd`.

Reply:
I think the previous state is more useful than just a 'unix' value. It may be too specific today, but I don't think we should jump to the other extreme.